domain/src/main/java/camp/nextstep/edu/github/GitHubInfoModel.kt

```diff
@@ -0,0 +1,7 @@
+package camp.nextstep.edu.github
+
+
+data class GitHubInfoModel(
```

Comment:
Nit: 개인적으로는 도메인 모델의 경우 최대한 나타내고자 하는 것 위주로 클래스 명을 지어주는 것이 더욱 좋습니다.
Model과 같은 네이밍은 큰 의미를 가지지 않습니다.
```suggestion
data class GithubRepositoryInfo(
```
그리고 github의 info가 아닌, githubRepository의 info가 더욱 어울리는 표현이겠네요!